client/blocks/login/error-notice.jsx

```diff
@@ -57,7 +57,7 @@ class ErrorNotice extends Component {
 	render() {
 		const error = this.getError();
 
-		if ( ! error || ( error.field && error.field !== 'global' ) || ! error.message ) {
+		if ( ! error || ( error.field && error.field !== 'global' ) || ! error.message || error.code === 'existing_wpcom_user' ) {
```

Comment:
I've split the error.code check into a separate condition, as it deserves a comment to explain what's happening.